docs/developer/migrate/from-ethereum.md

```diff
@@ -5,15 +5,7 @@ description: Overview of the similarities and differences between the Celo and E
 
 # Celo for Ethereum Developers
 
-Overview of the similarities and differences between the Celo and Ethereum blockchains.
-
----
```

Comment:
I'd keep the warning, as the rest of the page would need to be rewritten to accurately reflect Celo now.